lib/suites/LinkedDataSignature.js

```diff
@@ -50,6 +70,53 @@ module.exports = class LinkedDataSignature extends LinkedDataProof {
     this.useNativeCanonize = useNativeCanonize;
   }
 
+  /**
+   * See constructor docstring for param details.
+   *
+   * @returns {{verificationMethod: string, key: LDKeyPair, signer: Function,
+   *   verifier: Function}} - Validated and initialized key-related parameters.
+   */
+  _validateVerificationMethod({verificationMethod, key, signer, verifier}) {
```

Comment:
Since this is private, please move it to the bottom of the class. Possibly rename to something like `processSignatureParams` as well.

Reply:
Done (moved, renamed)